content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -230,15 +240,20 @@ clusters, zones, or regions.
 This model affords the following benefits:
 
 - Improved availability: If a control plane becomes unavailable, the scope of
-  the outage is limited to only that control plane.
+  the outage is limited to only that control plane, and the workloads in clusters managed by that control plane.
 
-- Configuration isolation: You can make configuration changes in one cluster,
-  zone, or region without impacting others.
+- Improved configuration isolation: Configuration changes in one control plane affects only
+the workloads in clusters managed by this control plane.
 
-You can improve control plane availability through fail over. When a control
-plane instance becomes unavailable, workload instances can connect to
-another available control plane instance. Fail over can happen across clusters,
-zones, or regions.
+#### Failover in Shared Control Planes
```

Comment:
Rephrase to just `Failover in Control Planes`

Reply:
as I said, I dont know if the failover works or is supported. It just seemed hand wavy. I can use replicated or shared. But its what it is.

Reply:
would be good to remove this before we have steps and validation.  Last time we discussed, i think mainly @costinm knows how to do this.